templates/layout.html

```diff
@@ -78,7 +83,32 @@
     <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/bootbox.js/5.5.2/bootbox.min.js" integrity="sha384-x8yg1bbOwuqjTf+DF/Sc3v14Tfzp2C61ThwPbDgflWBra3aBH4/vrB4z1WH3WwpP" crossorigin="anonymous"></script>
-
+    <script type="text/javascript" charset="utf-8">
```

Comment:
this should be inside the `scripts` block on just the template that requires it

Reply:
this still not addressed

Reply:
waiting on alex for this.